source_code/aux_mcu_v4/src/COMMS/comms_raw_hid.c

```diff
@@ -274,7 +294,15 @@ void comms_usb_communication_routine(void)
         {
             /* Reset flag */
             comms_raw_hid_packet_received[hid_interface] = FALSE;
-            
+
+            if (hid_interface == CTAP_INTERFACE)
+            {
+                uint8_t* usb_recast = (uint8_t*)&raw_hid_recv_buffer[hid_interface];
```

Comment:
same as above, I don't think it's the right cast

Reply:
same as above